### PR TITLE
feat(tutorial): restic backups using S3 - MTA-1203

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**
In this PR:
- new restic+S3 tutorial
- edits to bucket policy pages and menu

### CODE REVIEW

File: tutorials/restic-s3-backup/index.mdx [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+content:
+  h1: Creating backups with Restic and Object Storage
+  paragraph: This page shows how to create backups with Restic and Object Storage
+tags: restic-backup restic-s3 backup-s3 object-storage 
+categories:
+  - storage
+  - compute
+  - object-storage
+dates:
+  validation: 2022-03-31
```
Comment (RoRoJ) [inline tutorials/restic-s3-backup/index.mdx:14]:
```suggestion
  validation: 2022-04-04
```

File: tutorials/restic-s3-backup/index.mdx [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+
+1. Set up your [environment variables](https://github.com/scaleway/scaleway-sdk-go/tree/master/scw#environment-variables), if you have not done so yet. Replace `$SCW_ACCESS_KEY`, `$SCW_SECRET_KEY` and `$SCW_DEFAULT_PROJECT_ID` by their corresponding values.
+
+  ```
+  export SCW_ACCESS_KEY=$SCW_ACCESS_KEY
+  export SCW_SECRET_KEY=$SCW_SECRET_KEY
+  export SCW_DEFAULT_PROJECT_ID=$SCW_DEFAULT_PROJECT_ID
+  ```
+
+  Restic requires permission to perform the `PutObject` action on the bucket. In the next step we will implement a [bucket policy]() to grant the permissions.
```
Comment (RoRoJ) [inline tutorials/restic-s3-backup/index.mdx:70]: missing link

File: tutorials/restic-s3-backup/index.mdx [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+
+  </Message>
+
+  <Message type="tip">
+
+  Restic requests you to enter the password every time you perform actions on your repository. If you have configured automatic backups, you might want to set up an [environment variable](https://restic.readthedocs.io/en/latest/040_backup.html#environment-variables) for your password.
+
+  </Message>
+
+An output displays informing the name and path of your repository.
```
Comment (RoRoJ) [inline tutorials/restic-s3-backup/index.mdx:143]:
```suggestion
An output displays informing you of the name and path of your repository.
```

File: tutorials/restic-s3-backup/index.mdx [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+  Dirs:            1 new,     0 changed,     0 unmodified
+  Added to the repo: 732 B
+
+  processed 1 files, 22 B in 0:06
+  snapshot 15a39fd5 saved
+  ``` 
+
+  <Message type="tip">
+
+  Restic does not offer a built-in tool for automated backups. Restic recommends using tools like systemd and cron to schedule backup runs. Learn more on the [Restic documentation](https://restic.readthedocs.io/en/latest/040_backup.html#scheduling-backups).
```
Comment (RoRoJ) [inline tutorials/restic-s3-backup/index.mdx:187]:
```suggestion
  Restic does not offer a built-in tool for automated backups. Restic recommends using tools like systemd and cron to schedule backup runs. Learn more with the [Restic documentation](https://restic.readthedocs.io/en/latest/040_backup.html#scheduling-backups).
```

File: tutorials/restic-s3-backup/index.mdx (hunk 2 above)
Comment (ldecarvalho-doc) [reply tutorials/restic-s3-backup/index.mdx:70]:
```suggestion
  Restic requires permission to perform the `PutObject` action on the bucket. In the next step we will implement a [bucket policy](/storage/object/concepts/#bucket-policy) to grant the permissions. 
```

**Review by RoRoJ — APPROVED** [review]

File: tutorials/restic-s3-backup/index.mdx [hunk 5]
```diff
@@ -0,0 +1,232 @@
...
+
+A repository is the storage space where your backups will be hosted. In this tutorial, we will use Scaleway Object Storage buckets to host our backups.
+
+1. Set up your [environment variables](https://github.com/scaleway/scaleway-sdk-go/tree/master/scw#environment-variables), if you have not done so yet. Replace `$SCW_ACCESS_KEY`, `$SCW_SECRET_KEY` and `$SCW_DEFAULT_PROJECT_ID` by their corresponding values.
+
+  ```
+  export SCW_ACCESS_KEY=$SCW_ACCESS_KEY
+  export SCW_SECRET_KEY=$SCW_SECRET_KEY
+  export SCW_DEFAULT_PROJECT_ID=$SCW_DEFAULT_PROJECT_ID
+  ```
```
Comment (angristan) [inline tutorials/restic-s3-backup/index.mdx:68]: What's the purpose of this? We are not using the scw cli here

File: tutorials/restic-s3-backup/index.mdx [hunk 6]
```diff
@@ -0,0 +1,232 @@
...
+
+  </Message>
+
+3. Run the following command to apply the policy:
+
+  ```
+  aws s3api put-bucket-policy --bucket <bucket_name> --policy file://bucket-policy.json
+  ```
+
+  If the operation is successful, no output is returned.
```
Comment (angristan) [inline tutorials/restic-s3-backup/index.mdx:117]: What's the purpose of using a bucket policy? The credentials already have all the permissions required by restic

File: tutorials/restic-s3-backup/index.mdx [hunk 7]
```diff
@@ -0,0 +1,232 @@
...
+
+4. Initialize the repository using your bucket's endpoint as a backend. Your bucket endpoint can be found in the **Bucket Settings** tab under **Bucket Information**. 
+
+  ```
+  restic -r s3:https://s3.fr-par.scw.cloud/<bucket_name> init
+  ```
```
Comment (angristan) [inline tutorials/restic-s3-backup/index.mdx:123]: Aren't we missing the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` to use S3? cf https://restic.readthedocs.io/en/latest/080_examples.html#initializing-the-restic-repository

**Review by bene2k1 — APPROVED** [review]

File: tutorials/restic-s3-backup/index.mdx [hunk 8]
```diff
@@ -0,0 +1,182 @@
...
+  <Message type="tip">
+
+  Restic does not offer a built-in tool for automated backups. Restic recommends using tools like systemd and cron to schedule backup runs. Learn more on the [Restic documentation](https://restic.readthedocs.io/en/latest/040_backup.html#scheduling-backups).
+
+  </Message>
+
+3. Run the following command to verify that the backup was successful:
+
+  ```
+  aws s3api list-objects --bucket ahouseisnotahome
```
Comment (remyleone) [inline tutorials/restic-s3-backup/index.mdx:144]:
```suggestion
  aws s3api list-objects --bucket <bucket_name>
```

**Review by remyleone — APPROVED** [review]

**Pull request merged**